src/components/screens/inviteController.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import Home from "./home";
+import FriendsModal from "./inviteFriendsModal";
+import { InviteApi } from "../../services/inviteApi";
+
+function InviteController() {
+  const [showFriendsModal, setShowFriendsModal] = useState(false);
+  const [friends, setFriends] = useState([]);
+  const [message, setMessage] = useState("");
+  const inviteApi = new InviteApi(
+    process.env.REACT_APP_APIURL,
+    process.env.REACT_APP_APICODE
+  );
+
+  useEffect(() => {
+    inviteApi.getCandidates().then((response) => {
+      setFriends(response);
+    });
```

Comment:
You could add a .catch here to handle what happens if the request fails or the API is unavailable

```
    inviteApi.getCandidates().then((response) => {
      setFriends(response);
    }).catch((error) => {
      // handle the api returning an error or not responding as expected
      // possibly show an error message to the user
    })
```

You could also convert this to async by doing something like this:

```
  useEffect(() => {
    const handler = async () => {
      try {
        const response = await inviteApi.getCandidates();
        setFriends(response);
      } catch(e) {
        // handle the api returning an error or not responding as expected
        // possibly show an error message to the user
      }
    };
    handler()
  }, []);
```

Reply:
fixed w/ catch